src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/config.jelly

```diff
@@ -27,6 +27,10 @@
     <f:textbox/>
   </f:entry>
 
+  <f:entry field="retainedAfterTask" title="${%Retain slave after task}">
```

Comment:
instead of having a boolean, I think it's better to have `idleMinutes` here, like the docker plugin, so it can be set per template for how long it should be kept